examples/.scope/doctor-group-prompt.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: scope.github.com/v1alpha
+kind: ScopeDoctorGroup
+metadata:
+  name: prompt
+  description: Ask user if they want to run the fix
+spec:
+  include: when-required
+  actions:
+    - name: prompt-user
+      description: >-
+        Checks for an issue with a potentially destructive fix and
+        prompts the user for permission before running the fix.
+      check:
+        commands:
+          - sleep 5 #gives the progress bar time to display
+          - test -f .prompt-check
+      fix:
+        prompt:
+          text: |-
+            This may destroy some data.
+            Do you wish to continue?
+          # this is an optional field
+          helpText: >-
```

Comment:
In other parts of the schema, `helpText` refers to "humanized instructions to fix a problem", but that's quite different to this. What about a different name, like `extraContext`?

Reply:
I had the same concern. I like your suggestion. Will update the code.

Reply:
This is done.